packages/p2p/src/stores/order-store.js

```diff
@@ -280,6 +392,39 @@ export default class OrderStore {
         }
     }
 
+    @action.bound
+    verifyEmailVerificationCode(verification_action, verification_code) {
+        if (verification_action === 'p2p_order_confirm') {
+            requestWS({
+                p2p_order_confirm: 1,
+                id: this.order_id,
+                verification_code,
+                dry_run: 1,
+            }).then(response => {
+                this.setIsLoadingModalOpen(false);
+                if (response) {
+                    if (!response.error) {
+                        clearTimeout(wait);
+                        const wait = setTimeout(() => this.setIsEmailLinkVerifiedModalOpen(true), 600);
+                    } else if (
+                        (response.error.code === 'InvalidVerificationToken' ||
+                            response.error.code === 'ExcessiveVerificationRequests') &&
+                        // This is for an issue when both error codes are received simultaneously. DO NOT REMOVE.
+                        !response?.error.code === 'ExcessiveVerificationFailures'
+                    ) {
+                        clearTimeout(wait);
+                        this.setVerificationLinkErrorMessage(response.error.message);
+                        const wait = setTimeout(() => this.setIsInvalidVerificationLinkModalOpen(true), 700);
+                    } else if (response.error.code === 'ExcessiveVerificationFailures') {
+                        clearTimeout(wait);
+                        this.setVerificationLinkErrorMessage(response.error.message);
+                        const wait = setTimeout(() => this.setIsEmailLinkBlockedModalOpen(true), 600);
```

Comment:
Is there any difference of setting timeouts of `700` and `600` for `setVerificationLinkModalOpen` and `setIsEmailLinkBlockedModalOpen` ? Maybe we could use a constant for this, like `const TRANSITION_MODAL_DURATION = 700`

Reply:
It's just what works best for the modal and as for the constant, that would be too way many constants, let's not try that in this card, let's discuss it for another one.